bcipy/signal/processing/decomposition/psd.py

```diff
@@ -0,0 +1,80 @@
+import matplotlib.pyplot as plt
+import seaborn as sns
+import numpy as np
+from enum import Enum
+from scipy import signal
+from scipy.signal import welch
+from scipy.integrate import simps
+from mne.time_frequency import psd_array_multitaper
+
+
+class PSD_TYPE(Enum):
+    WELCH = 'Welch'
+    MULTITAPER = 'MutliTaper'
+
```

Comment:
The referenced blog article has some good points about why you might want to choose one PSD_TYPE over the other. It may be worth re-stating some of those points here or in the power_spectral_density docstring.